packages/core/src/editor/Block.css

```
@@ -453,6 +453,70 @@ NESTED BLOCKS
+/* Preview-source blocks (e.g. math) show their source in a popup, so they never
+get a native node selection. While selected they get this class instead, which
+we use to highlight the preview the same way `ProseMirror-selectednode` does. */
+.bn-preview-source-selected .bn-code-block-preview {
+  border-radius: 4px;
+  outline: 4px solid rgb(100, 160, 255);
+}
+.bn-code-block-source-popup {
+  position: absolute;
+  z-index: 1;
+
+  min-width: 240px;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)